### ISSUE TITLE
How to install the external apk into Island profile

### ISSUE THREAD

**Issue by himay03** [oasisfeng/island#454]
when I want to install the external apk download from source out of play store, the Island show the message "Sorry, this version variant of Island is not allowed to install (or clone) apps outside the system app store. Please refer to the FAQ for solutions."

What can I do to circumvent this issue? Thanks!

**Comment by GhettiGuru** [comment]
Just get the GitHub version and you can install whatever you want with the installer but even better so, try googling App Manager GitHub, it can with just wireless debugging, get app ops, spoof that stuff was installed from the play store which will allow Island to clone it, or you can manually go into App Manager settings, and click on the app and hit install for profile and pick one. With root access it'll ask you upon installation which user, but with wireless debugging it hasn't yet I've got to manually move it over there. Have fun.

**Comment by senghuo** [comment]
> when I want to install the external apk download from source out of play store, the Island show the message "Sorry, this version variant of Island is not allowed to install (or clone) apps outside the system app store. Please refer to the FAQ for solutions."
> 
> What can I do to circumvent this issue? Thanks!

copy apk to storage in Work profile. then open with Package installer instead of Island (app installer). it works for me.

**Comment by Ishaq12work00001** [comment]
Install any file manager apps from playstore in island then open file manager apps and search your external apps and install,
You must need sd card in your phone.

**Comment by viratnigam18** [comment]
> Just get the GitHub version and you can install whatever you want with the installer but even better so, try googling App Manager GitHub, it can with just wireless debugging, get app ops, spoof that stuff was installed from the play store which will allow Island to clone it, or you can manually go into App Manager settings, and click on the app and hit install for profile and pick one. With root access it'll ask you upon installation which user, but with wireless debugging it hasn't yet I've got to manually move it over there. Have fun.



> Just get the GitHub version and you can install whatever you want with the installer but even better so, try googling App Manager GitHub, it can with just wireless debugging, get app ops, spoof that stuff was installed from the play store which will allow Island to clone it, or you can manually go into App Manager settings, and click on the app and hit install for profile and pick one. With root access it'll ask you upon installation which user, but with wireless debugging it hasn't yet I've got to manually move it over there. Have fun.

How?

**Comment by viratnigam18** [comment]
> Install any file manager apps from playstore in island then open file manager apps and search your external apps and install, You must need sd card in your phone.

Did not work there is no SD card showing

**Comment by viratnigam18** [comment]
I just figured it out 

So just download island ext on both main and island 

https://github.com/oasisfeng/island/releases/tag/sideplay.v1.0  

And boom just clone your app and is work 🙈🥇

**Comment by keizah7** [comment]
> I just figured it out
> 
> So just download island ext on both main and island
> 
> https://github.com/oasisfeng/island/releases/tag/sideplay.v1.0
> 
> And boom just clone your app and is work 🙈🥇

can you help me? What do you mean "So just download island ext on both main and island"

Extension installing failing for me

**Comment by Mnstr3nrgy89** [comment]
> copy apk to storage in Work profile. then open with Package installer instead of Island (app installer). it works for me.

First off I see that this appears to be an old dead thread. But I just wanted to point out how stupidly simplictic this answer was. Everytime I saw package installer and island installer, I choose island because that's where I wanted to install it. Not sure why recently I started getting the error previously mentioned but it never occurred to me to tap on the Work Package Installer option. Worked just fine. Had to enable install from Yada Yada, but then it installed no problem.

-It's like hey I'm having a problem where I get these 2 options when trying to install and when I choose A it doesn't work.
-Have you tried selecting B to see what happens.
-No, let me give that a try..... OMG it worked like a charm.

Basically I feel like we had this conversation just now and I feel so stupid that i overlooked such an easy solution.

**Comment by Kyogre** [comment]
I just installed `FX File Explorer 9.1.0.0` (nextapp.fx) and used its feature named `Connect to storage` > select work profile > here copy apk you want to install > in apps drawer in `work profile` tab open `Files` app and install using default package manager > the app will be installed into the work profile.
Worked for me on Nothing phone 3a pro (android 16).

**Comment by owenmwen** [comment]
My Island app hijacked my normal package installer, so I cannot install anything at all, not even the extension

**Comment by viratnigam18** [comment]
> My Island app hijacked my normal package installer, so I cannot install anything at all, not even the extension
> 

I forgot what I was even doing at that time sorry guys 

What i might remember that I install app on the main phone side like we normally use
And the island one the fake or new one also installed the app


I don't know what app but yeah all i remember is this

I tried this on a samsung mobile 
Thanks 
Virat
